src/status_im/ui/screens/views.cljs

```diff
@@ -113,54 +113,79 @@
+          [react/main-screen-modal-view modal-view
+           [component]])]])))
+
 (defview main []
   (letsubs [signed-up? [:signed-up?]
-            view-id    [:get :view-id]
-            modal-view [:get :modal]]
+            view-id [:get :view-id]]
     {:component-will-update (fn [] (react/dismiss-keyboard!))}
     (when view-id
-      (let [component (case view-id
-                        :intro intro
-                        :create-account create-account
-                        :usage-data usage-data
-                        (:home :wallet :my-profile) main-tabs
-                        :browser browser
-                        :open-dapp open-dapp
-                        :dapp-description dapp-description
-                        :wallet-send-transaction send-transaction
-                        :wallet-transaction-sent transaction-sent
-                        :wallet-request-transaction request-transaction
-                        :wallet-send-transaction-request send-transaction-request
-                        (:transactions-history :unsigned-transactions) wallet-transactions/transactions
-                        :wallet-transaction-details wallet-transactions/transaction-details
-                        :wallet-send-assets wallet.components/send-assets
-                        :wallet-request-assets wallet.components/request-assets
-                        :new add-new
-                        :new-group new-group
-                        :add-contacts-toggle-list add-contacts-toggle-list
-                        :add-participants-toggle-list add-participants-toggle-list
-                        :edit-group-contact-list edit-contact-group-contact-list
-                        :new-public-chat new-public-chat
-                        :contact-toggle-list contact-toggle-list
-                        :new-chat new-chat
-                        :qr-scanner qr-scanner
-                        :chat chat
-                        :profile profile.contact/profile
-                        :group-chat-profile profile.group-chat/group-chat-profile
-                        :profile-photo-capture profile-photo-capture
-                        :accounts accounts
-                        :login login
-                        :recover recover
-                        :network-settings network-settings
-                        :network-details network-details
-                        :offline-messaging-settings offline-messaging-settings
-                        :currency-settings currency-settings
-                        :recent-recipients recent-recipients
-                        :recipient-qr-code recipient-qr-code
-                        :contact-code contact-code
-                        :profile-qr-viewer profile.user/qr-viewer
-                        :backup-seed backup-seed
-                        [react/view [react/text (str "Unknown view: " view-id)]])
+      (let [component (get-main-component view-id)
```

Comment:
Algned values? :)

Reply:
In Cursive you can configure this at Editor > Code Style > Clojure, General tab, "Align let-binding values" checkbox